generation/__tests__/ios.js

```diff
@@ -333,6 +333,28 @@ describe("iOS generation", () => {
 		});
 	});
 
+	describe("instance methods", () => {
```

Comment:
Those tabs look awful in github :(

Reply:
They look great. Don’t switch to spaces.

Reply:
🛒- i